Lesson3/Shopper/Shopper/Model/ShoppingModel.swift

```diff
@@ -0,0 +1,32 @@
+import Foundation
+
+class Shopping: ShoppingList {
+    var shoppingList: [String] = []
+    
+    func add(_ item: String) -> Bool {
+        if shoppingList.count == 10 {
+            return false
+        } else {
+            shoppingList.append(item)
+            return true
+        }
+    }
+    
+    func remove(_ item: String) -> Bool {
+        if shoppingList.contains(item) {
+            shoppingList.removeAll(where:{ item == $0 })
+            return true
+        } else {
+            return false
+        }
+    }
+    
+    func remove(at index: Int) -> Bool {
```

Comment:
Imagine the following:
```swift
remove(at: -1)
```

Reply:
changed the logic in the remove(at index:) method